api/pom.xml

```diff
@@ -209,6 +209,11 @@
 			<groupId>org.hibernate</groupId>
 			<artifactId>hibernate-validator</artifactId>
 	   </dependency>
+	   <dependency>
+			<groupId>mysql</groupId>
```

Comment:
Is this part of your changes?

Reply:
@dkayiwa no it is not

Reply:
Then why is it here?

Reply:
I don't know how it got mixed up with my PR but I see it's you @dkayiwa  and @reagan-meant  who did this commit.

Reply:
Did you create the branch off of master? Hope not on my trunk

Reply:
I created off master @reagan-meant

Reply:
@Irenyak1 you need to squash the commits into one

Reply:
@gitcliff yeah sure.